cmake/configure_lvi_mitigation_build.cmake

```diff
@@ -70,13 +70,13 @@ function (configure_lvi_mitigation_build)
         ${OE_BINDIR}/${C_COMPILER}
         PARENT_SCOPE)
   else ()
-    message(FATAL_ERROR "-- ${C_COMPILER} is not found.")
+    message(FATAL_ERROR "-- ${OE_BINDIR}/${C_COMPILER} is not found.")
```

Comment:
For the consistency, can you also update the message for the CXX compiler?

Reply:
Sure, now added on line 89.

Reply:
Thanks!